### DISCUSSION TITLE
Disable automatic scroll on append

### DISCUSSION THREAD

**Discussion by ng-babyshark** [MurhafSousli/ngx-scrollbar#437] [Q&A]
If we have a scrollable list and we append to it a new item, the scroll will automatically move to the bottom (as in the video). Is there a way to disable this behaviour? So that on append, the scroll will maintain current position. 

https://user-images.githubusercontent.com/89215669/162631760-ef4e6cf3-9f2d-4249-a547-5bc3ede25a9d.mov

**Comment by MurhafSousli** [comment]
This is not the default behavior, in this example it calls a function to scroll the bottom, as it shows in the source code

https://github.com/MurhafSousli/ngx-scrollbar/blob/47af53b20af79fb7d5170af8498a0b06d5279099/projects/ngx-scrollbar-demo/src/app/example2/example2.component.ts#L100-L102
